recipes/gtsam/all/conanfile.py

```diff
@@ -34,8 +36,7 @@ class gtsamConan(ConanFile):
                "build_type_postfixes": [True, False],
                "install_matlab_toolbox": [True, False],
                "install_cython_toolbox": [True, False],
-               "install_cppunitlite": [True, False],
-               "install_geographiclib": [True, False]}
```

Comment:
I do not think we are allowed to remote options

Reply:
no, they must be deprecated:
- add `"deprecated"` value in possible values of these options
- set default value to `"deprecated"`
- display a warning in `configure()` if value is different than `"deprecated"`
- delete these options from self.info in `package_id()`

Look at lz4 option in `librdkafka` recipe for a concrete example.

Reply:
I see. I didn't know that. 
After the CI finish this round of builds I'll push a fix for this.